src/main/java/net/pms/encoders/FFmpegAudio.java

```diff
@@ -201,8 +201,9 @@ public ProcessWrapper launchTranscode(
 		}
 
 		// Streaming support
-		cmdList.add("-movflags");
-		cmdList.add("+faststart");
+//		cmdList.add("-movflags");
+//		cmdList.add("frag_keyframe+empty_moov");
+		
```

Comment:
@atamariya That's a speedy one, more well supported as well ;-)

Is that "isml+frag_keyframe" will give same result ?